config/initializers/delayed_job.rb

```diff
@@ -0,0 +1,9 @@
+Delayed::Worker.destroy_failed_jobs = false
+Delayed::Worker.sleep_delay = 60
+Delayed::Worker.max_attempts = 3
+Delayed::Worker.max_run_time = 5.minutes
+Delayed::Worker.read_ahead = 10
+Delayed::Worker.default_queue_name = 'default'
+Delayed::Worker.delay_jobs = !Rails.env.test?
+Delayed::Worker.raise_signal_exceptions = :term
+Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
```

Comment:
I wonder if we can utilize the default logger so that it appears in STDOUT and is visible in our deployment environment (heroku). @m-pereira can you double check the recommendations for this?

Reply:
Nice, I'll check this, and change to default rails logger

Reply:
For test logger setting, I just add a `raise 'error test'` for a mailer, and call it with `deliver_later`.

This was my foreman log:

![image](https://user-images.githubusercontent.com/47258878/128226193-5b2fcb81-70ec-4bf8-84f3-a65ea0eed806.png)

@edwinthinks  Is it enough for heroku?

Reply:
@m-pereira i think this works :)! Thank you for checking